group_vars/all

```diff
@@ -4,6 +4,8 @@ app_package: flaskapp
 app_callable: app
 base_dir: '/srv/{{ app_name }}'
 nginx_server_name: 'localhost {{ app_name }}.local'
+nginx_http_port: 80
```

Comment:
In my case, I had to deploy on an in-house testing server that had Apache Tomcat already bound to port 80. I'd assume  a few people would run into the same situation. 
Ports 80 and 443 are set as safe defaults just incase no one needs it.